src/graphics/axis.hpp

```diff
@@ -0,0 +1,95 @@
+#ifndef AXIS_HPP
+#define AXIS_HPP
+
+#include "drawable.hpp"
+#include "esthetic.hpp"
+
+#include <SFML/Graphics/CircleShape.hpp>
+#include <SFML/Graphics/RectangleShape.hpp>
+
+namespace graphic {
+
+class Axis : public Drawable {
+
+  private:
+    sf::RectangleShape line;
+    sf::CircleShape    arrow;
+
+    float        size_arrow;
+    sf::Vector2f size_line;
+    sf::Vector2f position_line;
+    sf::Vector2f position_arrow;
```

Comment:
Should'nt the arrow position be calculated rather than stored ? It should be deduced from the line position and length

Reply:
I want to change that but in dedicated feature branch.. I want to check if we can work with vectorial draw...